playbooks/configure-apt-sources.yml

```diff
@@ -101,7 +101,18 @@
         dest: "/etc/openstack_deploy/group_vars/all/{{ item | basename }}"
       with_items:
         - files/apt.yml
-        - files/maas.yml
+      run_once: true
```

Comment:
Why change this to run_once? As these are fixed defaults which should not be edited, they should *always* be copied, otherwise how do we handle minor upgrades after release which include changes to these files?

Reply:
`run_once` is used so the task, which is delegated to the localhost, the deployment host, is only executed one time instead of iterating through all "hosts" and running on the delegated target.